templates/wafer/base.html

```diff
@@ -32,6 +32,10 @@ <h1>{{ WAFER_CONFERENCE_NAME }}</h1>
             <a href="https://www.offerzen.com/" target="_blank">
                 <img src="{% static 'img/logo_offerzen.png' %}"/>
             </a>
+            <br>
+            <a href="http://www.afrolabs.co.za/" target="_blank">
+                <img style="height: 200px" src="{% static 'img/logo_afrolabs.png' %}"/>
```

Comment:
Why do we specify a height here but not elsewhere? If it's too resize the image, could we not just resize the image and store the correct image in the repository instead of making browsers resize it?

Reply:
Fair point, I'll resize the image instead.